content/intermediate/operations/stacksets/index.md

```diff
@@ -0,0 +1,220 @@
+9. On **Review** page, review the contents of the page and choose **Submit**.
+10. Refresh the StackSet creation page until you see the **Status** of the **CREATE** operation as `SUCCEEDED`.
+11. Under **Stack instances**, you should see two stacks deployed. One in `us-east-1` and another in `us-west-2`.
+12. Navigate to **Exports**. You should see new entry with Export name `AWS-CloudFormationWorkshop-SecurityGroupId`.
+
+![StackSetsSecuritygroupexports](/static/intermediate/operations/stacksets/exportssecuritygroup.png)
+
+Congratulations! You have learned how export the output values from one stack set and import this value into another stack set.
+
+
+### Challenge
+
+In this exercise, you will use the knowledge gained from earlier parts of this lab to deploy a new stack set that would provision an [Amazon Elastic Compute Cloud (Amazon EC2)](https://docs.aws.amazon.com/ec2/?id=docs_gateway) instance in the existing VPC and attach the security group you created earlier. Your task is to update the template `example_ec2instance.yaml` to import `SubnetId1` that you exported as part of Part 1 of this lab. You will also choose to deploy in **Parallel**. The architecture diagram highlighting the EC2 instance you will describe as part of this challenge is shown next:
+
+![StackSetsEc2instance](/static/intermediate/operations/stacksets/stacksetsec2instance.png)
+
+:::expand{header="Need a hint?"}
+
+* Change directory to the `code/workspace/stacksets` directory.
+* Open the `example_ec2instance.yaml` CloudFormation template in the text editor of your choice.
+
+::alert[Note: [Amazon Machine Image (AMI) IDs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) are different for instances in different regions. To use region-specific AMI IDs, understand the following code snippet in the Parameters section of your template queries the latest AMI ID for that region and this is also referenced (`LatestAmiId`) in **Parameters** section of your template.]{type="info"}
+
+```yaml
+  LatestAmiId:
+    Description: The ID of the region-specific Amazon Machine Image to use.
+    Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
+    Default: /aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2
+```
+* Edit the Resources section of the template to import the value for SubnetId1 that was exported in Part 1. You can import the parameter of your choice the same way you imported VPC ID from `example_network.yaml` to `example_securitygroup.yaml`.
+:::
+
+:::expand{header="Want to see the solution?"}
+* You can find the full solution in the `code/solutions/stacksets/example_ec2instance.yaml` template.
+* To deploy stack sets in parallel, when creating a new stack set, in **Deployment Options**, choose to deploy as **Parallel**. This will deploy StackSets operation in both the regions in **parallel**, thus saving time.
```

Comment:
```suggestion
* Add the following line to the EC2 instance properties: `      SubnetId: !ImportValue AWS-CloudFormationWorkshop-SubnetId1`.
* Use the updated template, and create a new `example-ec2instance-workshop` stack set to deploy the EC2 instance resources in the 2 regions you chose earlier.  To deploy StackSets operations in parallel, from **Deployment Options** choose **Parallel**. This will deploy StackSets operations in both regions in parallel, thus saving time.
```